transit_odp/publish/templates/publish/snippets/disruptions_map.html

```diff
@@ -0,0 +1,45 @@
+<head>
+  <style>
+    .mapboxgl-popup-content {
+      font: 400 15px/22px "GDS Transport", arial, sans-serif;
+      padding: 0;
+      width: 300px;
+    }
+
+    .mapboxgl-popup-content h3 {
+      background: #005EA5;
+      color: #fff;
+      margin: 0;
+      padding: 10px;
+      border-radius: 6px 6px 0 0;
+      font-weight: 600;
+      margin-top: -15px;
+    }
+
+    .mapboxgl-popup-content div {
+      padding: 10px;
+      border-radius: 0 0 6px 6px;
+      font-weight: 700;
+      background: #ffffff;
+      border: 2px solid  #005EA5;
+    }
+
+    .mapboxgl-popup-content a {
+      border: none;
+    }
+
+    .mapboxgl-popup-anchor-top > .mapboxgl-popup-content {
+      margin-top: 15px;
+    }
+
+    .mapboxgl-popup-anchor-top > .mapboxgl-popup-tip {
+      border-bottom-color: #005EA5;
+    }
+
+    .mapboxgl-popup-anchor-bottom > .mapboxgl-popup-tip {
+      border-top-color: #005EA5;
+    }
+
```

Comment:
Can we reduce bottom padding on the information box?

Reply:
done